examples/resources/hcp_consul_cluster/federation.tf

```diff
@@ -1,19 +1,27 @@
-resource "hcp_hvn" "example" {
+resource "hcp_hvn" "east" {
```

Comment:
I'm not sure this example is getting rendered anywhere in the docs 🤔 I'd move this into `examples/guides/consul_cluster_federation/main.tf `